examples/session.example.ts

```diff
@@ -19,6 +19,7 @@ function fakeLoad() {
 
 (async () => {
   const ig = new IgApiClient();
+  let userInfo;
```

Comment:
This variable got no type. It is `implicit any`. Below you are writing different data types into this, it's a bad practice.